app/scripts/modules/google/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -449,6 +449,17 @@ module.exports = angular.module('spinnaker.serverGroup.configure.gce.configurati
 
         return result;
       };
+
+      command.zoneChanged = function zoneChanged() {
+        var result = { dirty: { } };
+        if(command.zone === undefined) {
```

Comment:
I believe this also has to consider `command.regional`. If the server group being configured is regional, no zone is required.